Buffs/LuluR/LuluR.cs

```diff
@@ -1,44 +1,45 @@
-﻿using System;
-using LeagueSandbox.GameServer.Logic.GameObjects;
-using LeagueSandbox.GameServer.Logic.Scripting;
+﻿using LeagueSandbox.GameServer.Logic.GameObjects;
 using LeagueSandbox.GameServer.Logic.API;
+using LeagueSandbox.GameServer.Logic.Scripting.CSharp;
 
 namespace LuluR
 {
-    class LuluR : BuffGameScript
+    class LuluR : IGameScript
     {
+        GameScriptInformation info;
+        Spell spell;
+        Unit owner;
+        Unit target;
         ChampionStatModifier statMod;
         float healthBefore;
         float meantimeDamage;
         float healthNow;
         float healthBonus;
-
-        public void OnActivate(Unit unit, Spell ownerSpell)
+        public void OnActivate(GameScriptInformation scriptInfo)
         {
+            info = scriptInfo;
+            spell = info.OwnerSpell;
```

Comment:
These should be implemented in a way that we don't need to re-declare them in every spell. Maybe add a property to a base class